src/App.tsx

```diff
@@ -1,9 +1,23 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+  const filteredMovies = moviesFromServer.filter(({ title, description }) => {
```

Comment:
create a visibleMovies variable containing filtered movies;

Reply:
point 2 in task

Reply:
Agreed, better rename, because this name is more meaningful.

[create a visibleMovies variable containing filtered movies](create a visibleMovies variable containing filtered movies;)